ape_etherscan/verify.py

```diff
@@ -44,68 +48,92 @@ class LicenseType(Enum):
 
     NO_LICENSE = 1
     """
-    Nobody else can copy, distribute, or modify your work without being at risk of
+    Nobody else can copy, distribute, or modify your work without risk of
     take-downs, shake-downs, or litigation.
+    https://github.com/github/choosealicense.com/blob/gh-pages/no-permission.md
     """
 
     UNLICENSED = 2
     """
-    A license with no conditions whatsoever which dedicates works to the public domain.
+    Unlicensed works, modifications, and larger works may be distributed
+    under different terms and without source code.
+    https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/unlicense.txt
```

Comment:
issue: This is not the same thing.  "UNLICENSED" is not the same as The Unlicense.  The SPDX identifier being `Unlicensed` for the former and `Unlicense` for the latter.  `Unlicensed` is actually technically not valid and not listed in the SPDX list.

In fact, this confusion [is not new, which we can see from this Stack Exchange discussion](https://opensource.stackexchange.com/questions/12410/is-there-a-difference-between-unlicense-org-and-spdx-unlicensed).

[The Solidity docs](https://docs.soliditylang.org/en/latest/layout-of-source-files.html#spdx-license-identifier) are more inline to what I thought `UNLICENSED` meant.  They state:

> If you do not want to specify a license or if the source code is not open-source, please use the special value UNLICENSED. Note that UNLICENSED (no usage allowed, not present in SPDX license list) is different from UNLICENSE (grants all rights to everyone).

Reply:
Here's [Etherscan's license reference](https://etherscan.io/contract-license-types) to add to the confusion.  Code `2` does in fact mean "The Unlicense".  If we're making these description modifiers here, I think it might be a good idea to update this var name to `UNLICENSE` or something to reduce confusion a bit.

Reply:
Yeah, I went back and forth and decided to split the difference… return 1 for “UNLICENSED” in the spdx identifier but use “NO_LICENSE” for the enum key as it’s more in line with the etherscan docs (since this is the Etherscan plugin)

Reply:
Ah, yeah the key for 2 should be “UNLICENSE” not “UNLICENSED”

Reply:
Oh I see what you mean, the key for 2 should be `UNLICENSE` not `UNLICENSED`

Reply:
Ah, looks like you added `unlicensed` key is correct in `_SPDX_ID_TO_API_CODE` and it maps the the correct Etherscan ID, I missed that on my first pass.  :+1: 

I guess just the Enum key should be `UNLICENSE` rather than `UNLICENSED` to reduce confusion.